util/json.c

```diff
@@ -43,7 +43,7 @@ skip_literal(const uint8_t * buf, const uint8_t * end)
 static const uint8_t *
 skip_string(const uint8_t * buf, const uint8_t * end)
 {
-	char ch;
+	uint8_t ch;
```

Comment:
I'm changing this variable because on lines 53 and 59, we assign it a value from `buf`, which is `uint8_t`.